CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 ### Added
 * Added support to expose simulcast configuration
+* Changed `maxSupportedVideoFrameRate` to 30
+* Changed `updateDeviceCaptureFormat` to match fps after resolution
```

Comment:
Need to move change log to the way top by adding a [Unreleased]. 
I would have the below, open for changes.
```
[Unreleased]
### Fixed
* [Demo] Correct frame rate in device selection view

### Changed
* Changed `updateDeviceCaptureFormat` to match fps after resolution
```